app/assets/stylesheets/base.css.scss

```diff
@@ -69,6 +69,14 @@ a {
     margin: 0 auto;
 }
 
+.hide-display {
+  display: none;
+}
+
+.inline-p {
```

Comment:
I personally prefer class names to be semantic, and style-related markup to be kept in CSS under those headers. In other words, this kind of class name is really no different than adding `style="display: inline;"` directly to the element. If you wanted to change it later, you'd also have to change the name of the class everywhere, which minimizes the usefulness of the functionality of classes. Instead, why not just put this styling into the `.qset-name-modal` style block? If that class is used elsewhere, you could be more specific and use a `p.qset-name-modal { ... }`. In any case, note that most of our class names are/should be semantic, not just a description of the style. We have strayed from that (there are always exceptions to the rule!), but just wanted to bring it up for your consideration.